xml/System.Runtime.CompilerServices/RuntimeHelpers.xml

```diff
@@ -1129,8 +1129,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>This method ensures that there is sufficient stack to execute the average Framework function. If there is not enough stack, then it returns false.</summary>
```

Comment:
```suggestion
        <summary>Ensures that there is sufficient stack to execute the average Framework function. If there is not enough stack, then it returns false.</summary>
```

Reply:
What do you mean by Framework? Can a different word be used? It is not very clear.